pages/docs/concepts/asyncapi document/payload-schema.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Payload Schema
+weight: 270
+---
+
+The payload schema defines a message's format, data types, and properties to ensure that the payload follows a specific structure and data format.
+
+You should use AsyncAPI Schema to handle complex messages and structured data models. Enabling consumers to gain insights into the structure and data types of the payload.
+
+Use npm or yarn to install the `@asyncapi/avro-schema-parser` package to enable Avro schema parsing in AsyncAPI.
```

Comment:
Why are we talking about Avro when defining a complex payload schema? 🤔 

In fact, if a schema is too complex, the best tool to use is JSON Schema (or the default AsyncAPI Schema Object, which is almost the same).

Reply:
also cc @derberg here for this question ⬆️

Reply:
Done, Fixed it for JSON schema